content/en/docs/Contribute/_index.md

```diff
@@ -0,0 +1,528 @@
+tests. The name and email you provide must match those found in your
+`git config`, and your git name and email must match those used for the CNCF
+CLA.
+
+## Contribute new content
+
+```mermaid
+flowchart LR
+    subgraph first[How to contribute]
+    direction TB
+       T[ ] -.-
+       A[Sign the CNCF CLA] --> D[Write docs in markdown<br>and build site with Hugo]
+       D[Write docs in markdown<br>and build site with Hugo] --- E[Push source to GitHub]
+       E --- G[Open a pull request]
+    end
+
+classDef grey fill:#dddddd,stroke:#ffffff,stroke-width:px,color:#000000, font-size:15px;
+classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class A,B,C,D,E,F,G,H grey
+class S,T spacewhite
+class first,second white
+```
+
+**_Figure - Contributing new content_**
+
+The previous figure presents the basic steps for new docs contributions.
+
+To contribute new content pages or improve existing content pages, open a pull
+request (PR):
+
+- If your change is small, or you're unfamiliar with Git, read
+  [Changes using GitHub](#changes-using-github) to learn how to edit a page.
+- If your changes are large, read [Work from a local fork](#fork-the-repo) to
+  learn how to make changes locally on your computer.
+
+### Changes using GitHub {#changes-using-github}
+
+If you're less experienced with Git workflows, here's an easier method of
+opening a pull request. Figure 1 outlines the steps and the details follow.
+
+```mermaid
+flowchart LR
+A([fa:fa-user New<br>Contributor]) --- id1[(open-telemetry/opentelemetry.io<br>GitHub)]
+subgraph tasks[Changes using GitHub]
+direction TB
+    0[ ] -.-
+    1[1. Edit this page] --> 2[2. Use GitHub markdown<br>editor to make changes]
+    2 --> 3[3. fill in Propose file change]
+
+end
+subgraph tasks2[ ]
+direction TB
+4[4. select Propose file change] --> 5[5. select Create pull request] --> 6[6. fill in Open a pull request]
+6 --> 7[7. select Create pull request]
+end
+
+id1 --> tasks --> tasks2
+
+classDef grey fill:#dddddd,stroke:#ffffff,stroke-width:px,color:#000000, font-size:15px;
+classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
+classDef k8s fill:#326ce5,stroke:#fff,stroke-width:1px,color:#fff;
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class A,1,2,3,4,5,6,7 grey
+class 0 spacewhite
+class tasks,tasks2 white
+class id1 k8s
+```
+
+Figure 1. Steps for opening a PR using GitHub.
+
+1. On the page where you see the issue, select the **Edit this page** option in
+   the right-hand side navigation panel.
+
+1. Make your changes in the GitHub editor.
+
+1. Below the editor, fill in the **Propose file change** form.
+
+1. Select **Propose file change**.
+
+1. Select **Create pull request**.
+
+1. The **Open a pull request** screen appears. Your description helps reviewers
+   understand your change.
+
+1. Select **Create pull request**.
```

Comment:
I am very certain that this flow also does not work if someone is not member of the otel gh org, there is a step where github will ask you about forking.

Reply:
GH implicitly creates a fork when doing a small edit from the UI. I'll add that.